mobile/src/core/theme/colors.ts

```diff
@@ -13,3 +13,14 @@ export enum Colors {
   RED = '#FF0000',
   GREEN = '#00FF00'
 }
+
+export const AccountColors = [
+  Colors.GREY_LIGHT,
+  Colors.GREY_DARK,
+  Colors.WHITE,
+  Colors.BLUE_LIGHT,
+  Colors.BLUE_DARKEST,
+  Colors.BLACK,
+  Colors.BLUE,
+  Colors.BLUE_DARKER
```

Comment:
This currently means we run out of colors after 8 accounts, will use modulo to probably wrap them around.

Reply:
You might use a color scheme generator